data/json/recipes/weapon/magazines.json

```diff
@@ -79,6 +103,7 @@
     "skills_required": [ "gun", 2 ],
     "time": "8 m",
     "autolearn": true,
+    "tools": [ [ [ "nail", -1 ], [ "combatnail", -1 ] ] ],
```

Comment:
Wouldn't this need guns that actually use the magazine? Those are nail rifle and coilgun at the moment.

Reply:
done.

Reply:
done